### PR TITLE
Add Stickermule Sponsor

### PR DESCRIPTION
**Pull request by MrNickCasey**
Stickermule is sponsoring cuHacking with promo codes, and credits. This adds their logo with a link to $10 promo code.

### CODE REVIEW

File: src/assets/sponsors/stickermule.svg [hunk 1]
```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (FemmeQueenInSTEM) [inline src/assets/sponsors/stickermule.svg]: This SVG needs to be minified.

**Review by walsquared — CHANGES_REQUESTED** [review]
Close this PR, and make a new one that requests to merge changes to master. We need this out immediately.

File: src/assets/sponsors/stickermule.svg (hunk 1 above)
Comment (MrNickCasey) [reply src/assets/sponsors/stickermule.svg]: Addressed in latest commit.

File: src/pages/home/index.js [hunk 2]
```diff
@@ -66,25 +67,25 @@ function Sponsors() {
...
+        <div className={styles.logoSection}> 
           {sponsorLogo(styles.marchNetworks, 'https://marchnetworks.com', MarchNetworks)}
         </div>
-        <div className={styles.logoSection}>
+        <div className={styles.logoSection}> 
           {sponsorLogo(styles.balsamiq, 'https://balsamiq.com', Balsamiq)}
           {sponsorLogo(styles.fdmGroup, 'https://fdmgroup.com', FdmGroup)}
         </div>
-        <div className={styles.logoSection}>
+        <div className={styles.logoSection}>
```
Comment (FemmeQueenInSTEM) [inline src/pages/home/index.js]: I'm not sure how you managed to add white space here, but please remove it (for my sake, it's annoying).

**Review by walsquared — APPROVED** [review]
Looks (almost) good to me, thanks for being prompt.

File: src/pages/home/index.js (hunk 2 above)
Comment (MrNickCasey) [reply src/pages/home/index.js]: Not sure how that got there either. Addressed in latest commit.

**Pull request merged**